### PR TITLE
Fix printing planemo test logs

### PR DESCRIPTION
**Pull request by mvdbeek**
We were quitting the context manager scope too early, so the print threads would stop too early.
I think this is the last fix after which I would create a new planemo release.

### CODE REVIEW

**Review by jdavcs — APPROVED** [review]

**Comment by martenson** [comment]
the failure may be related

**Comment by mvdbeek** [comment]
Yep, had to use the with statement in cmd_serve.py as well.

File: planemo/commands/cmd_serve.py [hunk 1]
```diff
@@ -40,4 +40,5 @@ def cli(ctx, uris, **kwds):
     paths = uris_to_paths(ctx, uris)
     runnables = for_paths(paths)
     kwds["galaxy_skip_client_build"] = kwds.pop("skip_client_build", False)
-    galaxy_serve(ctx, runnables, **kwds)
+    with galaxy_serve(ctx, runnables, **kwds) as config:
+        pass
```
Comment (martenson) [inline planemo/commands/cmd_serve.py:44]:
```suggestion
        yield config
```

File: planemo/commands/cmd_serve.py [hunk 2]
```diff
@@ -40,4 +40,5 @@ def cli(ctx, uris, **kwds):
     paths = uris_to_paths(ctx, uris)
     runnables = for_paths(paths)
     kwds["galaxy_skip_client_build"] = kwds.pop("skip_client_build", False)
-    galaxy_serve(ctx, runnables, **kwds)
+    with galaxy_serve(ctx, runnables, **kwds) as config:
```
[the remark this answers is not in the document]
Comment (mvdbeek) [reply planemo/commands/cmd_serve.py]: Yep, don't need config here.

**Pull request merged**